packages/react/src/component-tests/IcLoadingIndicator/IcLoadingIndicatorTestData.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { ReactElement } from "react";
+import { IcLoadingIndicator } from "../../components";
+
+export const IndeterminateCircularLoadingIndWithLabel = (): ReactElement => (
+  <IcLoadingIndicator label="Loading..."></IcLoadingIndicator>
+);
+
+export const DeterminateCircularLoadingIndWithLabel = (args: {
+  progress: number;
+  min: number;
+  max: number;
+}): ReactElement => (
+  <IcLoadingIndicator
+    progress={args.progress}
+    min={args.min}
+    max={args.max}
+    label="Loading..."
+  ></IcLoadingIndicator>
+);
+
+export const DeterminateCircularLoadingIndWithLabelSizeSmallDefaultLarge =
+  (args: { progress: number; min: number; max: number }): ReactElement => (
+    <div style={{ display: "grid", gap: "50px" }}>
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Small"
+        size="small"
+      />
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Default"
+      />
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Large"
+        size="large"
+      />
+    </div>
+  );
+
+export const DeterminateCirclularLoadingIndWithIcon = (args: {
+  progress: number;
+  min: number;
+  max: number;
+}): ReactElement => (
+  <IcLoadingIndicator
+    size="icon"
+    progress={args.progress}
+    min={args.min}
+    max={args.max}
+  />
+);
+
+export const DeterminateLinearLoadingIndWithLabel = (args: {
+  progress: number;
+  min: number;
+  max: number;
+}): ReactElement => (
+  <IcLoadingIndicator
+    progress={args.progress}
+    min={args.min}
+    max={args.max}
+    label="Loading..."
+    type="linear"
+  />
+);
+
+export const DeterminateLinearLoadingIndWithLabelSizeSmallDefaultLargeFull =
+  (args: { progress: number; min: number; max: number }): ReactElement => (
+    <div style={{ display: "grid", gap: "50px" }}>
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Small"
+        size="small"
+        type="linear"
+      />
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Default"
+        type="linear"
+      />
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Large"
+        size="large"
+        type="linear"
+      />
+      <IcLoadingIndicator
+        progress={args.progress}
+        min={args.min}
+        max={args.max}
+        label="Full-width"
+        size="default"
+        type="linear"
+        fullWidth={true}
+      />
+    </div>
+  );
+
+export const DeterminateCircularDarkLoadingIndWithLabel = (args: {
```

Comment:
I think the name of this is misleading as it's actually applying appearance light
```suggestion
export const DeterminateCircularLightLoadingIndWithLabel = (args: {
```